### DISCUSSION TITLE
Discussions vs Issues

### DISCUSSION THREAD

**Discussion by cobaltt7** [ScratchAddons/ScratchAddons#1152] [Help] [answered]
What's the difference between Feature/New addon tags on Issues and the Ideas tag on Disscusions?

**Comment by cobaltt7** [comment]
bump

**Reply by TheColaber** [reply]
Disscusions are really used for help and for signing up for translating. Issues are for ideas, bugs, and new addons.

**Reply by cobaltt7** [reply]
@TheColaber well why is there an ideas tag in discussions then

**Reply by GrahamSH-LLK** [reply, marked as answer]
Because you _can_ have ideas in discussions, but if you want it implemented, you should make an issue.

**Reply by GrahamSH-LLK** [reply]
Also, this isn't quite a forum, you don't need to bump.

**Reply by ghost** [reply]
@RedGuy12 The Discussions tag has been moved to Discussions. Also, Discussions is more like a "community knowledge base" if you ask me.

_It looks like this is a question. If this answered your question, please mark it as the answer._
